src/python/WMCore/Cache/WMConfigCache.py

```diff
@@ -267,12 +272,12 @@ def loadByID(self, configID):
         """
         try:
             self.document = self.database.document(id=configID)
-            if 'owner' in self.document.keys():
+            if 'owner' in list(self.document):
```

Comment:
Doesn't the `in` operator work in both py2 and py3 (without the need to cast to list)? Like
`if 'owner' in self.document:`